miso-web/src/main/webapp/pages/editRun.jsp

```diff
@@ -303,7 +303,7 @@
     </div>
   </c:if>
 
-  <c:if test="${not empty run.metrics}">
+  <c:if test="${run.health.key ne 'Failed' and run.health.key ne 'Stopped' and metrixEnabled and miso:instanceOf(run, 'uk.ac.bbsrc.tgac.miso.core.data.IlluminaRun')}">
```

Comment:
I would think that there are, and have made a ticket to investigate that. This fix, however, reverts to display the Interop Metrics that were available before.